test/go.sum

```diff
@@ -173,8 +173,8 @@ github.com/mailru/easyjson v0.7.7 h1:UGYAvKxe3sBsEDzO8ZeWOSlIQfWFlxbzLZe7hwFURr0
 github.com/mailru/easyjson v0.7.7/go.mod h1:xzfreul335JAWq5oZzymOObrkdz5UnU4kGfJJLY9Nlc=
 github.com/mattn/go-isatty v0.0.20 h1:xfD0iDuEKnDkl03q4limB+vH+GxLEtL/jb4xVJSWWEY=
 github.com/mattn/go-isatty v0.0.20/go.mod h1:W+V8PltTTMOvKvAeJH7IuucS94S2C6jfK/D7dTCTo3Y=
-github.com/metal3-io/baremetal-operator/apis v0.9.0 h1:D/2II9H4/+ldz5JO+zufBO5VqYjhrJlg89dFgWMKdoE=
-github.com/metal3-io/baremetal-operator/apis v0.9.0/go.mod h1:gNPRYPdH/IjTHkmdyFWNLlpWvT+hyfjHH+9CEkdLcWY=
+github.com/metal3-io/baremetal-operator/apis v0.9.1 h1:Qj0ALDQMIBGebDHhiKXvWs8Hve8KObCwUIlSgbozQ5A=
+github.com/metal3-io/baremetal-operator/apis v0.9.1/go.mod h1:D0NG169s4/Nt/TT//DVq/DdgfB23my/SqXbwTgIo198=
 github.com/metal3-io/baremetal-operator/pkg/hardwareutils v0.9.0 h1:4xilUV03Z2OPs2QzGQvm/rg1+ptROc2f0pZg+nzNsRA=
```

Comment:
let's do it manually and do hardwareutils bump too.

Reply:
Sure, I'll do that too.